Sources/ScipioKit/ProjectGenerator/BuildSettingsGenerator.swift

```diff
@@ -81,6 +81,7 @@ struct ProjectBuildSettingsGenerator {
             "GCC_PREPROCESSOR_DEFINITIONS": [.inherited, "SWIFT_PACKAGE=1"],
             "USE_HEADERMAP": false,
             "CLANG_ENABLE_OBJC_ARC": true,
+            "SKIP_INSTALL": true,
```

Comment:
If my understand is correct, SKIP_INSTALL is false to build as xcframework, why it is set as true explicitly?

Reply:
Your point is good. I'm confused about `SKIP_INSTALL` usage. As I investigated, `SKIP_INSTALL` is overwritten on `xcodebuild` command.
https://github.com/giginet/Scipio/blob/main/Sources/ScipioKit/XcodeBuild/Commands/ArchiveCommand.swift#L26

They should be more explicit.
I'll fix this.